### PR TITLE
Improve codespaces instructions

### PR DESCRIPTION
**Pull request by Luke-Oldenburg**
## Summary of the problem
The previous instructions informed users to use docker on codespaces. However, docker is not preinstalled and directly running `./bin/dev` is much easier.

## Describe your changes
I changed the instructions to inform users to run `./bin/dev` instead of continuing on to follow the docker instructions. I also included tips on how to fix the port forwarding if it breaks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Please email [hcb-security@hackclub.com](mailto:hcb-security@hackclub.com) to re
 
 [GitHub Codespaces](https://docs.github.com/en/codespaces) allows you to run a development environment without installing anything on your computer, allows for multiple instances, creates an overall streamlined and reproducible environment, and enables anyone with browser or VS Code access to contribute.
 
-To get started, [whip up a codespace](https://docs.github.com/en/codespaces/getting-started/quickstart) and follow the steps in the [Automated Setup with Docker](#automated-setup-with-docker) section.
+To get started, [whip up a codespace](https://docs.github.com/en/codespaces/getting-started/quickstart) and then run `./bin/dev`. If you are having issues with opening the link that is posted in the terminal, try making the `3000` port public under the `PORTS` tab in your terminal
```
Comment (sampoder) [inline README.md]:
```suggestion
To get started, [whip up a codespace](https://docs.github.com/en/codespaces/getting-started/quickstart) and run `bin/dev`. If you are having issues with opening the link that is posted in the terminal, ensure the `3000` port is public under the `PORTS` tab in your terminal.
```

**Review by sampoder — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ Please email [hcb-security@hackclub.com](mailto:hcb-security@hackclub.com) to re
 
 [GitHub Codespaces](https://docs.github.com/en/codespaces) allows you to run a development environment without installing anything on your computer, allows for multiple instances, creates an overall streamlined and reproducible environment, and enables anyone with browser or VS Code access to contribute.
 
-To get started, [whip up a codespace](https://docs.github.com/en/codespaces/getting-started/quickstart) and follow the steps in the [Automated Setup with Docker](#automated-setup-with-docker) section.
+To get started, [whip up a codespace](https://docs.github.com/en/codespaces/getting-started/quickstart) and run `bin/dev`. If you are having issues with opening the link that is posted in the terminal, ensure the `3000` port is public under the `PORTS` tab in your terminal.
```
Comment (sampoder) [inline README.md]:
```suggestion
To get started, [whip up a codespace](https://docs.github.com/en/codespaces/getting-started/quickstart) and run `bin/dev`. If you are having issues with opening the link that is printed in the terminal, ensure the `3000` port is public under the `PORTS` tab in your terminal.
```

**Review by sampoder — APPROVED** [review]
LGTM! Just made some small adjustments to the writing style

File: README.md (hunk 2 above)
Comment (Luke-Oldenburg) [reply README.md]: 3000 port doesn't _have_ to be public which is why I wrote "try making".

**Pull request merged**